src/components/Routes.tsx

```diff
@@ -200,6 +201,22 @@ export function Routes() {
           </DataFetcher>
         )}
       />
+      <AuthenticatedRoute
+        path="/lab/probe_hybridisation_qc"
+        render={(routeProps) => (
+          <DataFetcher
+            key={routeProps.location.key}
+            dataFetcher={() =>
+              stanCore.GetComments({
+                commentCategory: 'section',
```

Comment:
This should not be 'section', but Probe hybridisation comments I hope

Reply:
Good point, as the comments are applied for section, I thought we need to get the section comments?do  you think we need tp get the Probe QC comment instead ?